hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/record/SingleTransactionRecordBuilderImpl.java

```diff
@@ -243,7 +251,10 @@ public SingleTransactionRecord build() {
         final Timestamp consensusTimestamp = HapiUtils.asTimestamp(consensusNow);
         final Timestamp parentConsensusTimestamp =
                 parentConsensus != null ? HapiUtils.asTimestamp(parentConsensus) : null;
-
+        // sort the automatic associations to match the order of mono-service records
+        automaticTokenAssociations.sort(
+                Comparator.<TokenAssociation>comparingLong(a -> a.tokenId().tokenNum())
+                        .thenComparingLong(a -> a.accountIdOrThrow().accountNum()));
```

Comment:
Can we find a single place to use this instead of spreading it across different files?

Reply:
Declared as private static variable in same file. 
Do you have any recommendations where to move it ? It is just used in this file afaik 🤔 . Other one was `FcTokenAssociation`

Reply:
Oh I didn't realize these were different classes. One has `Fc` in front 😇 my mistake!